astropy/modeling/core.py

```diff
@@ -2212,6 +2221,36 @@ class FittableModel(Model):
     col_fit_deriv = True
     fittable = True
 
+    def __call__(self, *args, **kwargs):
+        if not args:
+            # Inputs were passed as keyword (not positional) arguments.
+            # Because the signature of the ``__call__`` is defined at
+            # the class level, the name of the inputs cannot be changed at
+            # the instance level and the old names are always present in the
+            # signature of the method. In order to use the new names of the
+            # inputs, the old names are taken out of ``kwargs``, the input
+            # values are sorted in the order of self.inputs and passed as
+            # positional arguments to ``__call__``.
+
+            # these are the keys that are always present as keyword arguments
+            keys = ['model_set_axis', 'with_bounding_box', 'fill_value',
+                    'equivalencies', 'inputs_map']
+
+            new_inputs = {}
+            # kwargs contain the names of the new inputs + ``keys``
+            allkeys = list(kwargs.keys())
+            # Remove the names of the new inputs from kwargs and save them
+            # to a dict ``new_inputs``.
+            for key in allkeys:
+                if key not in keys:
+                    new_inputs[key] = kwargs[key]
+                    del kwargs[key]
+            # Create positional arguments from the keyword arguments in ``new_inputs``.
+            args = []
+            for k in self.inputs:
+                args.append(new_inputs[k])
```

Comment:
What about the case where someone uses both positional and keyword arguments? E.g., the original input names are x and y, and they rename them to b and c and then call f(1.2, b=7.1). It would seem that in this case the conversion of the keyword argument to positional won't happen or is that incorrect?

Reply:
You are right, this won't work. I will push a fix shortly.